lib/core/src/sdk.rs

```diff
@@ -234,10 +245,34 @@ impl LiquidSdk {
             receive_swap_handler,
             chain_swap_handler,
             buy_bitcoin_service,
+            external_input_parsers,
         });
         Ok(sdk)
     }
 
+    fn get_all_external_input_parsers(config: &Config) -> Option<Vec<ExternalInputParser>> {
+        let external_input_parsers = if config.use_default_external_input_parsers {
```

Comment:
Shouldn't we concatenate the user external input parser on top of the default if both the `use_default_external_input_parsers` is true and the user provides additional parsers?
Just thinking atm these config settings are mutually exclusive exclusive.

Reply:
That was the idea :) I've refactored and it should be clearer now [015823c](https://github.com/breez/breez-sdk-liquid/pull/614/commits/015823c8b5e1ed95db60c25ee86d34e4f267eea0).